pom.xml

```diff
@@ -100,18 +100,15 @@
         <commons-pool2.version>2.12.0</commons-pool2.version>
         <commons-lang3.version>3.14.0</commons-lang3.version>
         <cglib.version>2.2.2</cglib.version>
-        <mockito.version>4.11.0</mockito.version>
         <curator.version>5.6.0</curator.version>
-        <zookeeper.version>3.9.1</zookeeper.version>
+        <mockito.version>4.11.0</mockito.version>
         <jackson.version>2.14.2</jackson.version>
```

Comment:
We can remove the jackson pom import and all related jackson dependencies.

Reply:
Removed.